packages/ui/pages/index.tsx

```diff
@@ -69,6 +71,10 @@ const Home: NextPage = () => {
         {/* <SuggestionOverview report={data.report} /> */}
 
         <Grid.Container gap={2} justify={'center'}>
+          <Grid sm={12} md={3}>
+            <testPlugin.cardView suggestions={data.report.suggestions} />
+          </Grid>
```

Comment:
We need to re-work this a little more.

Reply:
Talking about this offline, we have to have the suggestions be scoped to the plugin.